.github/workflows/cicd.yml

```
@@ -0,0 +1,370 @@
+  #         status: ${{ job.status }}
+  #       env:
+  #         SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+    deploy-pr:
+    if: github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'synchronize' || github.event.action == 'reopened' || github.event.action == 'labeled')
+    name: (ephemeral staging) Deploy
+    runs-on: ubuntu-latest
+    #needs: [build-test-push]
+    environment: staging
+    steps:
+      - name: "Check if any label matches *.ghost.is"
```

actionlint (1.7.4)

"steps" job is sequence node but mapping node is expected

(syntax-check)